src/Microsoft.Azure.EventHubs/Primitives/ClientInfo.cs

```diff
@@ -23,7 +23,7 @@ static ClientInfo()
                 product = GetAssemblyAttributeValue<AssemblyProductAttribute>(assembly, p => p.Product);
                 version = GetAssemblyAttributeValue<AssemblyVersionAttribute>(assembly, v => v.Version);
                 framework = GetAssemblyAttributeValue<TargetFrameworkAttribute>(assembly, f => f.FrameworkName);
-#if NETSTANDARD
+#if NETSTANDARD1_3
```

Comment:
This is build specific. If we move to another build of NetStandard this will be broken. I wonder if there is a preprocessor that refers all netstandard versions.